docs/upgrade/upgrade-7.17-8.x.asciidoc

```diff
@@ -0,0 +1,155 @@
+[[upgrade-7.17-8x]]
+== Upgrade from 7.17 to an 8.x version
+
+[float]
+=== Why it's important to upgrade
+
+Upgrading provides you access to {elastic-sec}'s latest features, enhancements, and bug fixes, many of which enable you to save your company money, respond faster to potential threats, investigate, and analyze data with enhanced tooling. For more benefits, check out our blog, https://www.elastic.co/blog/top-5-reasons-to-upgrade-elastic-security[Top 5 reasons to upgrade Elastic Security]. Also, it's important to ensure your deployment is fully maintained and supported. For more information, refer to https://www.elastic.co/support/eol[Elastic's Product End of Life Dates]. 
+
+[float]
+=== Plan for your upgrade
+
+Before upgrading to {elastic-sec} 8.x, consider the following recommendations:
+
+* Plan for an appropriate amount of time to complete the upgrade. Depending on your configuration and the size of your cluster, the process can take up to a week to complete.
+
+* Open a https://support.elastic.co[support case] with Elastic to alert our Customer Support team of your system change. You can also get support by purchasing a https://www.elastic.co/consulting[consulting engagement]. 
```

Comment:
This should be mentioned only if the customer has a subscription which grants consultative support.

Reply:
++ Agreed.  Perhaps "If your support subscription includes consultative support..."